integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -0,0 +1,52 @@
+package com.redhat.service.bridge.integration.tests.steps;
+
+import java.time.Duration;
+
+import com.redhat.service.bridge.integration.tests.common.Utils;
+import io.cucumber.java.en.And;
+import io.restassured.RestAssured;
+import org.awaitility.Awaitility;
+
+import com.redhat.service.bridge.integration.tests.context.TestContext;
+import com.redhat.service.bridge.integration.tests.context.resolver.ContextResolver;
+
+
+import io.cucumber.java.en.Then;
+import org.hamcrest.Matchers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class WebhookSiteSteps {
+
+    private static final String ENDPOINT_BASE_URL = "https://webhook.site/";
+
+    //    private static final String ENDPOINT_UUID = "a0704e8f-a817-4d02-b30a-b8c49d0132dc";
+    private static final String ENDPOINT_UUID = Utils.getSystemProperty("webhook.site.uuid");
+
+    private TestContext context;
+
+    public WebhookSiteSteps(TestContext context) {
+        this.context = context;
+    }
+
+    @And("^Webhook site request history is cleared$")
```

Comment:
This step can cause a race condition (i.e. running 2 PR checks at the same time).
One possibility could be to rather implement a hook checking all requests from site and deleting all requests older than ~24 hours, what do you think?

Reply:
@sutaakar I tried to implement your suggestion as https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/494/commits/d315c82743c7a66aab9d093ab85efa0b3796c10f , please let me know your opinion?

Reply:
Implementation looks ok.
Can you move it to `Hooks` class and make a hook out of it, being executed before test suite (`@BeforeAll`)?

Reply:
should be there now